README.md

```diff
@@ -247,7 +246,7 @@ create-or-update-tags \
 --tags ResourceId=my-auto-scaling-group,ResourceType=auto-scaling-group,Key=spot-enabled,Value=true,PropagateAtLaunch=false
 ```
 
-**Note**
+##### Note #####
```

Comment:
This note isn't added to the TOC, which could be fine, but would be problematic if using automatic TOC generators.